init_scripts/init-user.sh

```diff
@@ -1,11 +1,19 @@
-#!/bin/sh
-# init-user.sh - Initialize user-level requirements for our project
-# Should be set in the Vagrantfilee to run once with low privileges (non-root).
-
-# Using pipenv as our default python package manager
-# Also, we specify the version to avoid possible inconsistency
-# with future releases, and --user to reduce global fingerprint
-# on the system as a whole.
-python -m pip install -I pipenv==2020.8.13 --user
-cd /vagrant/sellotape && python -m pipenv sync
-echo "-----2ND INIT DONE-----"
+#!/bin/sh
+# init-user.sh - Initialize user-level requirements for our project
+# Should be set in the Vagrantfilee to run once with low privileges (non-root).
+
+# Add aliases to the vagrant user
+echo ". ~/.aliases.sh" >> ~/.bashrc
+chmod +x ~/.aliases.sh
+
+# Using pipenv as our default python package manager
+# Also, we specify the version to avoid possible inconsistency
+# with future releases, and --user to reduce global fingerprint
+# on the system as a whole.
+python -m pip install -I pipenv==2020.8.13 --user

```

Comment:
you should probably put `python3` here to ensure you're always running the right python version